bot/rules/everyone_ping.py

```diff
@@ -0,0 +1,37 @@
+import textwrap
+from typing import Dict, Iterable, List, Optional, Tuple
+
+from discord import Embed, Member, Message
+
+from bot.constants import Colours
+
+
+async def apply(
+    last_message: Message,
+    recent_messages: List[Message],
+    config: Dict[str, int],
+) -> Optional[Tuple[str, Iterable[Member], Iterable[Message]]]:
+    """Detects if a user has sent an '@everyone' ping."""
+    relevant_messages = tuple(msg for msg in recent_messages if msg.author == last_message.author)
+
+    ev_msgs_ct = 0
+    for msg in relevant_messages:
+        if "@everyone" in msg.content:
+            ev_msgs_ct += 1
+
+    if ev_msgs_ct > config["max"]:
+        # Send the user an embed giving them more info:
+        embed_text = textwrap.dedent(
+            f"""
+            Please don't try to ping {last_message.guild.member_count:,} people.
+            **It will not have good results.**
```

Comment:
I don't like the `It will not have good results` part of this, that feels like we're saying it's okay to ping 95000 people but it won't get you the results you were hoping for.

Instead of this second line, maybe we can give this embed a headline. For example, you could use a random headline from `bot.constants.NEGATIVE_REPLIES`?

Reply:
With the second line removed, this can just be a regular f string. There's no need for textwrap dedent.

Reply:
By `It will not have good results`, I was trying to say that people would get upset if you pinged them.

I'll look at `bot.constants.NEGATIVE_REPLIES`, didn't know that existed.